e2e-tests/src/ProjectTest.js

```diff
@@ -77,11 +77,11 @@ export const flow = {
         },
         {
             "type": "navigate",
-            "url": "http://localhost:3000/",
+            "url": "https://deploy-preview-67--sporosdaoapp-dev.netlify.app/",
             "assertedEvents": [
                 {
                     "type": "navigation",
-                    "url": "http://localhost:3000/",
+                    "url": "https://deploy-preview-67--sporosdaoapp-dev.netlify.app/",
```

Comment:
Same comment as above. Do not use hardcoded URLs. The test is supposed to cover the latest code in the PR.